Subject: [PATCH] From c084c8349d1780980e232cb28b60a109e3d89438 Mon Sep 17
 00:00:00 2001 From: Koen Kooi <koen@dominion.thruhere.net> Date: Wed, 2 Nov
 2011 16:54:57 +0100 Subject: [PATCH] Upstream-Status: Pending

update patch to version 7.4.4
Signed-off-by: Changqing Li <changqing.li@windriver.com>
---
 ext/imap/config.m4 | 10 ++--------
 1 file changed, 2 insertions(+), 8 deletions(-)

diff --git a/ext/imap/config.m4 b/ext/imap/config.m4
index 5086a31..0e938bd 100644
--- a/ext/imap/config.m4
+++ b/ext/imap/config.m4
@@ -110,7 +110,7 @@ if test "$PHP_IMAP" != "no"; then
     PHP_NEW_EXTENSION(imap, php_imap.c, $ext_shared,, -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1)
     AC_DEFINE(HAVE_IMAP,1,[ ])
 
-    for i in $PHP_IMAP /usr/local /usr; do
+    for i in $PHP_IMAP $PHP_IMAP/usr /usr/local /usr; do
       IMAP_INC_CHK()
       el[]IMAP_INC_CHK(/include/c-client)
       el[]IMAP_INC_CHK(/include/imap)
@@ -199,13 +199,7 @@ if test "$PHP_IMAP" != "no"; then
       AC_MSG_ERROR(Cannot find rfc822.h. Please check your c-client installation.)
     fi
 
-    if test ! -r "$IMAP_DIR/c-client/libc-client.a" && test -r "$IMAP_DIR/c-client/c-client.a" ; then
-      ln -s "$IMAP_DIR/c-client/c-client.a" "$IMAP_DIR/c-client/libc-client.a" >/dev/null 2>&1
-    elif test ! -r "$IMAP_DIR/$PHP_LIBDIR/libc-client.a" && test -r "$IMAP_DIR/$PHP_LIBDIR/c-client.a"; then
-      ln -s "$IMAP_DIR/$PHP_LIBDIR/c-client.a" "$IMAP_DIR/$PHP_LIBDIR/libc-client.a" >/dev/null 2>&1
-    fi
-
-    for lib in c-client4 c-client imap; do
+    for lib in /usr/lib c-client4 c-client imap; do
       IMAP_LIB=$lib
       IMAP_LIB_CHK($PHP_LIBDIR)
       IMAP_LIB_CHK(c-client)
-- 
2.7.4

